src/components/light.js

```diff
@@ -11,6 +11,23 @@ var CubeLoader = new THREE.CubeTextureLoader();
 
 var probeCache = {};
 
+function distanceOfPointFromPlane (positionOnPlane, planeNormal, p1) {
```

Comment:
Maybe factor out into [utils/math.js](https://github.com/aframevr/aframe/tree/master/src/utils) to keep this tidier.